tasks/gitlab_helpers.py

```diff
@@ -10,7 +10,9 @@
 
 from tasks.libs.common.color import Color, color_message
 
-CI_VISIBILITY_URL = "https://app.datadoghq.com/ci/pipeline-executions"
+CI_VISIBILITY_URL = "https://app.datadoghq.com/ci"
+PIPELINE_VISIBILITY_URL = f"{CI_VISIBILITY_URL}/pipeline-executions"
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement. -->
**🥜 nitpick**
The product name is CI Visibility vs CI Test Visibility for tests